src/test/java/de/tum/in/www1/artemis/notification/SingleUserNotificationServiceTest.java

```diff
@@ -457,6 +457,7 @@ void testConversationNotificationsNewMessageReply() {
 
         singleUserNotificationService.notifyUserAboutNewMessageReply(answerPost, user, userTwo);
         verify(messagingTemplate, times(1)).convertAndSend(eq("/topic/user/" + user.getId() + "/notifications"), (Object) any());
+        verify(generalInstantNotificationService, times(1)).sendNotification(any(), (User) any(), any());
```

Comment:
#6749 mentions that the usage of `any()` with `verify()` calls may lead to flakiness. Is it possible to avoid using `any()` somehow?

Reply:
Yes I removed it.